recipes/service.rb

```diff
@@ -0,0 +1,39 @@
+#
+# Cookbook: rundeck-bridge
+# Recipe:   service
+#
+
+# Compute final command with every options (even from Wrapper cookbooks)
+cmd = [ node['rundeck_bridge']['binary'] ]
+node['rundeck_bridge']['options'].each do |key, value|
+ cmd << "--#{key} #{value}"
+end
+command = cmd.join(' ')
+
+
+# Overwrite and update the command
+# (Workaround for Upstart and logs)
+node['rundeck_bridge']['poise_service']['options'].each do |k, v|
```

Comment:
IMO, I would rather have hacks in wrapper cookbooks and have this one as proper as possible.